elastic/kibana
# [Remote Clusters] Show what type of security model each cluster is using

## Summary

This PR enhances the current UI to highlight the auth method used by the remote clusters in the list in flyout.

##### How to test

* Start up kibana and es
* Navigate to Stack management -> Remote clusters and create a new remote cluster
* Edit /common/lib/cluster_serialization.ts:117 to be just `api_key` or `certificate` in order to override the state in the UI.
* Verify that the selected auth method is shown in the table and flyout.


<details>
<summary>Screenshots</summary>

// Certificates
![Screenshot 2023-12-18 at 08 52 40](https://github.com/elastic/kibana/assets/1191206/e8b97119-b5fa-43aa-85c5-b90b65ecd060)
![Screenshot 2023-12-18 at 08 52 45](https://github.com/elastic/kibana/assets/1191206/a21854a4-1f55-4ca9-ab9e-c5892fdfe04d)
// Api key
![Screenshot 2023-12-18 at 08 54 57](https://github.com/elastic/kibana/assets/1191206/edd1022a-db51-4ec9-bc5b-8e472b3e4d67)
// Details flyout
![Screenshot 2023-12-18 at 08 52 51](https://github.com/elastic/kibana/assets/1191206/3a583716-680b-48d9-81ff-f07b87ab7e8a)


</details>
